packages/components/avatar/src/avatar.ts

```diff
@@ -161,18 +175,21 @@ export class Avatar extends LitElement {
       } else {
         this.initials = this.profileName = '';
       }
+
+      this.#checkOverflow();
     }
   }
 
   override render(): TemplateResult {
     return html`
       ${this.image ? this.#renderPicture() : nothing}
+      <sl-tooltip id="avatar-tooltip-${this.#avatarId}">${this.profileName}</sl-tooltip>
```

Comment:
Since we're in Shadow DOM, there is no need to use a unique number, right? The id will already be unique *inside the Shadow DOM*?